Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:3.8-buster
```

Comment:
What's the purpose of this PR? To provide a sample Docker container demonstrating how to use Atheris?

If so, could you add some documentation at the top of the file and mention it in the README?

Reply:
The purpose is to build https://hub.docker.com/r/google/atheris.